src/textual/cli/cli.py

```diff
@@ -21,14 +23,27 @@ def run():
 
 
 @run.command(help="Run the Textual Devtools console.")
+@click.option(
+    "--port",
+    "port",
+    type=int,
+    default=None,
```

Comment:
Can the default not be an integer? So it shows in the help.

Reply:
I don't understand. The default is _not_ an integer (it is `None`).
If I change the default to an integer, it still doesn't show when I run `textual console --help`.

Reply:
I thought it would show the default. That's disappointing. In that case, would you mind updating the help text with the default?

Reply:
👍